config/api.yml

```diff
@@ -801,6 +801,17 @@
       :post:
       - :name: delete
         :identifier: cloud_volume_snapshot_delete
+  :vm_infras:
+    :description: Vm Infras
+    :identifier: vm_infra
+    :options:
+    - :collection
+    :verbs: *gpppd
+    :klass: Vm
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: vm_infra_view
```

Comment:
I was expecting a reconfigure action to be added to the vms section, not an entirely new collection/controller